.github/images/entrypoint.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -e
+
+echo -e "\n\n--------- Starting the configuration of rskj ---------\n\n"
+cd /usr/src/
+git clone https://github.com/rsksmart/rskj.git rskj
+cd rskj && git checkout ${INPUT_RSKJ_BRANCH}
+chmod +x ./configure.sh && chmod +x gradlew
+./configure.sh
+
+echo -e  "\n\n--------- Starting the configuration of powpeg ---------\n\n"
+cd /usr/src/
+git clone https://github.com/rsksmart/powpeg-node.git powpeg
+cp configure_gradle_federator.sh powpeg
+cd powpeg && git checkout ${INPUT_FEDERATOR_BRANCH}
+chmod +x ./configure.sh && chmod +x gradlew
+FED_VERSION=$(bash configure_gradle_federator.sh)
+echo "FED_VERSION=$FED_VERSION"
+./configure.sh
+./gradlew  --info --no-daemon clean build -x test
+
+echo -e "\n\n--------- Starting the configuration of RIT ---------\n\n"
+cd /usr/src/
+git clone https://github.com/rsksmart/rootstock-integration-tests.git rit
+mv configure_rit_locally.sh rit
+mv regtest.js rit/config/regtest.js
+mv /usr/src/logbacks/* /usr/src/rit/logbacks/
+cd rit
+git checkout ${INPUT_RIT_BRANCH}
+chmod +x ./configure.sh
+./configure.sh
+./configure_rit_locally.sh "${FED_VERSION}"
+export LOG_LEVEL=${INPUT_RIT_LOG_LEVEL}
```

Comment:
This is an untrusted input. As such, please quote it.

```suggestion
export LOG_LEVEL="${INPUT_RIT_LOG_LEVEL}"
```

Or use it in 2 steps, which gives the same guaruantees as quoting:

```
LOG_LEVEL=${INPUT_RIT_LOG_LEVEL}
export LOG_LEVEL
```